scripts/e2e-test.sh

```diff
@@ -78,3 +78,24 @@ helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.image.tag=dev \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
 
+helm template deploy/helm/eunomia-operator/ \
```

Comment:
please add a comment above explaining that below block is testing the hello-world-yaml example